Cargo.toml

```diff
@@ -6,6 +6,7 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-reqwest = { version = "0.11.12", features = ["json", "blocking"] }
+reqwest = { version = "0.11", default-features = false, features = ["blocking", "json", "rustls-tls-native-roots"] }
```

Comment:
I switched it to rustls since openssl failed to build (probably don't have the necessary libs installed, didn't really want to check). I guess you might want to switch it back, openssl will probably give you a smaller binary and if it's linked in dynamically, it can be updated separately by the OS. But rustls is generally more pain-free to build ig.